config/samples/cxx_example_loadtest_with_pre_built_workers.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: cxx-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: cxx
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+    - language: cxx
+      run:
+        image: gcr.io/grpc-testing/wanlin/pre_built_workers/cxx
```

Comment:
`gcr.io/grpc-testing/wanlin/pre_built_workers/cxx` seems to only be the prefix of the actual image name that will be used (since I'll basically need a new image for each revision of grpc). is `gcr.io/grpc-testing/wanlin/pre_built_workers/cxx` just a placeholder value here? What's going to be the actual value and how will it get injected?

Reply:
Yes, `testing/wanlin/pre_built_workers/cxx` is just a place holder, it should be replaced by ${pre_built_server_image_registry} or ${pre_built_client_image_registry}. This example load test configuration yaml file will then become a template for cxx load test example using pre-built image. `tools/run_tests/performance/loadtest_template.py` (in Paulo's pr 25976) will gather such templates from all languages to generate a big template with all languages. This step is not run by users, it should be part of the infrastructure. 

When using the pre-built workers, the workflow should be like:
1. Call  `prepare_for_prebuilt_workers.go` with language, preBuiltImagePrefix (where they want to push these images to). The `prepare_for_prebuilt_workers.go` will build, push the images to that repository and write this repository information to a file at location specified by `REBUILD_WORKER_OUTPUT_PATH`.  
2. Call `loadtest_config.py` with information (including where we pull the images, this information is saved n a file from previous step) need to be substituted. Then we will have a bunch of run-able tests.
3. The the load test yaml configuration files will be then be applied to cluster to run.   

It is a little complicate, let me know if there is anything I did not explain well or you have any concern with.